pkg/epp/server/runserver.go

```diff
@@ -80,6 +81,7 @@ const (
 	DefaultCertPath                                 = ""                               // default for --cert-path
 	DefaultConfigFile                               = ""                               // default for --config-file
 	DefaultConfigText                               = ""                               // default for --config-text
+	DefaultMetricsStalenessThreshold                = 2 * time.Second
```

Comment:
@liu-cong wasn't this originally 5 seconds (when it was named metricsValidityPeriod)?

Reply:
Originally there is no threshold enforced (though we highlight metrics older than 5s in logs, never saw that happen though). This PR doesn't enforce the threshold either. But a followup PR is expected to enforce the threshold